platform/android/export/export.cpp

```diff
@@ -1455,6 +1483,86 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		}
 	}
 
+	Vector<uint8_t> _resize_launcher_icon(const Ref<Image> &p_source_image, const LauncherIcon p_icon) {
+		Ref<Image> working_image = p_source_image;
+
+		if (p_source_image->get_width() != p_icon.dimensions || p_source_image->get_height() != p_icon.dimensions) {
+			working_image = p_source_image->duplicate();
+			working_image->resize(p_icon.dimensions, p_icon.dimensions, Image::Interpolation::INTERPOLATE_LANCZOS);
+		}
+
+		PoolVector<uint8_t> png_buffer;
+		Error err = PNGDriverCommon::image_to_png(working_image, png_buffer);
+		if (err == OK) {
+			Vector<uint8_t> ret;
+			ret.resize(png_buffer.size());
+			memcpy(ret.ptrw(), png_buffer.read().ptr(), ret.size());
+			return ret;
+		} else {
+			String err_str = String("Failed to convert resized icon (") + p_icon.export_path + ") to png.";
```

Comment:
Should the Error value `err` be included in the message?

Reply:
I copied this error convention from the `_process_launcher_icons` method ([line 1480](https://github.com/amanj120/godot/pull/11/files#diff-fb88394f2df1faf844deaff3b3639aeeR1480)). In that method, the error value is not included in the message.

Reply:
That's fair. If it could be useful, I'd suggest including it. (I'm not sure what values are possible here.)